src/front-end/angular/components/tile.component.ts

```diff
@@ -4,11 +4,27 @@ import {
   Input,
 } from '@angular/core';
 
+import {
+  tileByNumber,
+} from '../../styles';
+
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'tile',
-  template: `{{ value }}`,
+  template: `
+  <style>
+    :host {
+      display: flex;
+      width: 100%;
+    }
+  </style>
+  <div [ngClass]="value === 0 ? emptyTile : tileByNumber(value)"></div>`,
```

Comment:
I moved this logic into this component to be consistent with the other frameworks. I had to add the :host settings to keep the tile the right size. it's not ideal but it's easier to keep the frameworks all in sync this way